src/frontend/ui/tiles/Assets.svelte

```diff
@@ -108,73 +116,82 @@
 
     return url;
   }
+
```

Comment:
What is happening with this sequence of reactive statemnts here? Cuase I believe you can just use
```js
let cacheIds = projectStore.getReactiveProjectCacheIds();
```